python/runtime/feature_derivation/feature_derivation.py

```diff
@@ -0,0 +1,630 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
```

Comment:
maybe keep the package strucure: `runtime.feature.derivation`?

Reply:
Great idea! Would rename in the next PR.